chain/addons/loki/copy.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
`#! /usr/bin/env bash`  will be better, it will find `bash` command in `PATH` env variables, while `#! /bin/bash` is an absolute path style, which maybe changed in some systems.